src/components/Layout/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, ReactNode} from "react"
+import { IFrameRendererProps, MultiLinkLayerProps } from "./types"
+import { useLoadedIframe } from "@/hooks/useLoadedIframe"
+import { Spinner } from "../Spinner"
+import { SetURLSearchParams } from "react-router-dom"
+
+export const MultiLinkLayer = ({ title, linkToArray,setSearchParams}: MultiLinkLayerProps) => {

```

Comment:
can we have a rounded border and a drop shadow to make the games div render a bit smoother?

Reply:
I have implemented it. Most of the game div is able to adapt it however 
```
<TimeTriviaFrame />
```

does not adapt to it prob due to its provider